desktop/src/main/java/bisq/desktop/primary/main/content/trade/TradeView.java

```diff
@@ -30,11 +30,12 @@ public TradeView(TradeModel model, TradeController controller) {
 
         headlineLabel.setText(Res.get("trade"));
 
-        addTab(Res.get("trade.overview"), NavigationTarget.TRADE_OVERVIEW);
-        addTab(Res.get("offerbook"), NavigationTarget.OFFERBOOK);
-        addTab(Res.get("openOffers"), NavigationTarget.OPEN_OFFERS);
-        addTab(Res.get("pendingTrades"), NavigationTarget.PENDING_TRADES);
-        addTab(Res.get("closedTrades"), NavigationTarget.CLOSED_TRADES);
+        addTab(Res.get("list"), NavigationTarget.TRADE_OVERVIEW);
+        addTab(Res.get("grid"), NavigationTarget.TRADE_OVERVIEW_GRID);
+//        addTab(Res.get("offerbook"), NavigationTarget.OFFERBOOK);
```

Comment:
Commented this out because [design](https://xd.adobe.com/view/88ec6f20-15ca-4326-81f4-30a30cc1aa0a-ddbf/screen/d354915a-8bc0-4e6a-916b-e58fff3e8117) contained two tabs (list / grid) - is it okay?

Reply:
Yes